### ISSUE TITLE
Change descripion for designers alternative text tip

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-quick-start#297]
[Comment received](https://lists.w3.org/Archives/Public/wai-eo-editors/2015Oct/0002.html) from Jordan Wilson <Jordan.Wilson@theprimacy.com>.

> I don't find this to be a useful design tip:
> [Provide alternative text for images](http://www.w3.org/WAI/gettingstarted/tips/designing.html#provide-alternative-text-for-images)
> I believe that's a very useful content (copywriter) tip and a very useful developer tip. I've found that for designers this just gets ignored because its not part of their job role. Instead I convey it differently:
>
> Non-text content (imagery, sound, video) need an appropriate text-alternative
> When your design includes imagery, sound or video consider how it will be conveyed to a user unable to perceive visible or audible content. Video or sounds will need a transcript, captions or audio descriptions. The design should provide a way to access those resources. Consider if your visual assets are informational (ex. important imagery, images of text or a chart) or decorative (ex. a separator or 'smiling businessperson' photography). Communicate with writers and developers to convey those needs.

**Comment by iadawn** [comment]
Suggested response, this would need appropriate topping and tailing, but this is the core content:

Thank you for your suggestion. Creating advice for designers on this topic is certainly challenging, and I think your approach adds important additional information and presents the task in a more realistic light.

I have taken the liberty of rewording your suggestion slightly to avoid the first person reference (a stylistic choice in all other tips) and to tighten up the delivery a little. I hope you are comfortable with the suggested edits.

> Consider how imagery, sound, or video will be accessed by users unable to perceive visual or audio content. The design should provide a way to access video or audio transcripts, captions, or audio descriptions.  Imagery may be informational (for example, a chart), functional (for example, icons and image buttons), or decorative (for example, 'smiling person' photography). Communicate with writers and developers to convey those needs.

**Comment by gheake-1** [comment]
I agree with this edit suggestion and the rewording to fit with the style of the tips.

**Comment by James-Green** [comment]
I think your reworded edit is great.  It's much more designer-centric.

**Comment by AndrewArch** [comment]
Well drafted Kevin - supported :)

**Comment by davidberman** [comment]
I also support Kevin's approach.

**Comment by slhenry** [comment]
comment on potential change https://github.com/w3c/wai-quick-start/commit/61a6cae

**Comment by iadawn** [comment]
The approach to this was discussed in [EO Meeting of 4th Dec](http://www.w3.org/2015/12/04-eo-minutes.html#item03) in light of Shawn's comment.

There was support for broadening the tip to address what to consider when adding non-text content to designs. There were two suggestions for changes to tip title:

> Provide text alternatives for images and media
> Provide text alternatives for non-text content

These and other possible options will need to be revisited.

Shawn also expressed some concern over the content. The following is a suggested rewrite that keeps to the spirit of the initial suggestion.

> Some text alternatives are provided visually, for example, links to transcripts, alternatives for icons, and captions for tables. Consider which non-text items required alternative text and how it will be displayed in your design. For text alternatives that are not visible, consider the purpose of the image or media and provide text which describes that purpose. For example, images might convey information, provide decoration, or support functionality. Alternative text is not needed for purely decorative images.

**Comment by nitedog** [comment]
Thanks Kevin.

It seems to me that we are trying to convey two slightly different messages in one tip:

1. Provide text alternatives for images
2. Ensure the design can accommodate alternatives to non-text content

My hope is that (1) is at least implicitly addressed through (2), and does not need to be spelled out as directly (because it is distracting to try to say two messages in one blurb). This needs agreement from EOWG, as it is a change to the objective of the key message for this tip. If EOWG does not agree with this change, then I think we need to decide if we want (2) to be a new separate tip.

In any case, my suggested rewrite for (2), which to me also implicitly addresses (1) would be this:
Title: Provide for alternatives for images and media
Blurb: Ensure that your design can accommodate alternatives for images and media that need to be provided visibly in the content. This includes links to transcripts for audio, text accompanying icons and buttons, and captions and descriptions for tables and complex graphs. Coordinate with the content authors and developers to determine the type of alternatives needed for non-text content, and how to best accommodate it in your designs.
Rationale: The second half of your blurb ("For text alternatives that are not visible...") seems to be tangential to the core message. If this is important, it should be included in one or more of the other "provide text alternatives" tips elsewhere in the resource. Also note the change from "provide text alternatives" to "provide *for* alternatives" to emphasize the design aspect of this particular tip.

**Comment by slhenry** [comment]
+1 to Shadi's version above

**Comment by davidberman** [comment]
+1 from me too

On 2015-12-08 10:31, shawn_slh wrote:
>
> +1 to Shadi's version above
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/w3c/wai-quick-start/issues/297#issuecomment-162915778>.
>

-- 
David Berman, RGD, FGDC LinkedIn <http://www.linkedin.com/in/bermandavid> Twitter @davidberman <http://www.twitter.com/davidberman> Facebook <http://www.facebook.com/davidbberman> Skype davidberman.com <skype:davidberman.com?chat> Google Plus 
<https://plus.google.com/u/0/+DavidBermanCom/posts>
David Berman Communications | berman@davidberman.com | @davidberman | blog <http://www.designedgecanada.com/author/david-berman>
+1-613-728-6777 | 340 Selby Avenue, Ottawa K2A 3X6

High Level Advisor, United Nations | GDC ethics chair | Ico-D Sustainability chair | Carleton University Access Network chair
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
/Accessibility courses:/ Vancouver | Victoria | Dublin | Ottawa | Toronto
/Upcoming:/ North Bay | Toronto | Mexico City | Cyprus | Bahrain
Enjoy this accessibility video we helped produce for CMA <http://ow.ly/TDy90> | Do Good book news: <http://www.dogoodbook.com/> "Don't just do good design ... do good!"

This message may contain proprietary information. Unauthorized disclosure/copying/distribution of contents prohibited.

**Comment by iadawn** [comment]
One thing to note regarding your identified 'what this tip is doing':

1. Provide text alternatives for images
2. Ensure the design can accommodate alternatives to non-text content

Of the two, if I recall, 1. is what Vicki expressed as important.

Just thinking about your wording.

**Title**

I appreciate and agree with the rationale, I just can't get my head around the actual words. I think it sounds overly complex, a bit too formal, and 'provide for' has a meaning of 'feed' or 'nourish'. However, every time I try to think of an alternative I end up pushing it more towards the second message you indicated much more. So far I have tried:

* Consider alternatives for images and media (possibly too weak)
* Include alternatives for images and media (probably too strong)
* Think about alternatives for images and media (may be a bit informal)

**Blurb**

I have made some minor tweaks to simplify the language a little:

> Ensure that your design can accommodate necessary visible alternatives for images and media. For example, links to transcripts of audio, text with icons and buttons, and captions and descriptions for tables or complex graphs. Work with content authors and developers to determine what alternatives are needed for non-text content, and how to best include it in your designs.

Thoughts?

**Comment by nitedog** [comment]
Yes, (1) is the current focus and (2) is what I understand is the newly requested focus. Either EOWG accepts this change from (1) to (2), confirms that a new tip addressing (2) should be added, or declines (2) altogether. I just don't think we should mix both (1) and (2) in one tip because it is too confusing. I do not recall this decision being discussed as clearly with EOWG. That was my point.

Regarding the title, I take your point. However, the three options you provide seem to speak about the alternatives themselves ("consider alternatives", "include alternatives", and "think about alternatives"), rather than the impact of visible alternatives on the design. Please work a little more on finding titles that better match (2).

As to the new blurb wording, I think it looks good. It just needs a good title, then EOWG can decide if this is a suitable replacement for the current tip, or if it needs to be a new tip.

**Comment by slhenry** [comment]
I strongly agree not to mix 1 & 2 in a single tip.
I support changing the old tip to address 2, which was the essence of the public comment.
I'm guessing that @Vicki and others will be fine with it, since it still mentions alts for images (and we have them more explicitly in the other pages) -- let's check with them asap.

Suggestion:
<blockquote>
<strong>Design for image and media alternatives</strong>

Ensure that your design provides alternatives for images and media as needed. For example, you might need visible links to transcripts of audio or links to audio described versions of video, text with icons or buttons, or captions and descriptions for tables or complex graphs. Work with content authors and developers to provide alternatives for non-text content.
</blockquote>

Rationale: "accommodate" can have negative connotations. Your "For example" sentence was not a complete sentence.

*Example?* Probably we do *not* need an example for this. However, an idea for an example is a video with underneath it links to: _Transcript_ and _Video with audio descriptions_.

**Comment by nitedog** [comment]
Title and wording work for me. If possible, a simple example would be helpful. I think the link to transcript idea is simple enough. Another idea would be a caption for a table or link to an inline description for a complex graph (no need to include the description - just the link is crucial).

**Comment by iadawn** [comment]
Just to capture the possible examples from above:

* Media player with visible transcript link (my preference as it highlights another issue of accessibility that needs to be considered - transcripts)
* Icon with alternative text showing
* Simple table with a caption below

These examples may benefit from a 'good' and 'bad' version to highlight the difference.

**Comment by yatil** [comment]
I still don’t like that a very specific tip is transformed to something that is much less specific. I especially think that the innuendos here provide too much overload for people who are getting started:

> Ensure that your design provides alternatives for images and media _as needed_. For example, you _might need_ visible links to transcripts of audio or links to audio described versions of video, text with icons or buttons, or captions and descriptions for tables or complex graphs. Work with content authors and developers to provide alternatives for non-text content.

I think a lot of people would just throw in the towel and say “How am I supposed to know when alternatives for images are needed? Do I need visible links to transcripts of audio or other stuff? Where is that explained to me? I am just starting out!”

I feel strongly that we should revert to the original tip and provide _an additional_ tip when we decide that we want to cover other kinds of text alternatives. And then provide hands-on suggestions for those.

**Comment by slhenry** [comment]
I do see your point... however, I think most of these tips are that way. That is, for pretty much all of these Tips, I think you need to follow the links to learn more. Unlike Easy Checks where we provide all the info you need, these Tips just provide a brief intro then you have to follow the links to really figure out what you need to do.

I do like that this focuses specifically on the designing task, whereas the previous tip overlapped a lot with writing & developing tasks.

I do think we need to consider if we now need to link to additional info about other alternatives. All the media SC seem like too much but maybe we link to http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv.html

**Comment by davidberman** [comment]
I agree with, at very least, linking to link to http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv.html .

**Comment by yatil** [comment]
> Ensure that your design provides alternatives for images and media as needed. Examples for such text alternatives are:
>
> * visible links to transcripts of audio
> * links to audio described versions of video
> * text with icons or buttons
> * captions and descriptions for tables or complex graphs
>
> Work with content authors and developers to provide alternatives for non-text content.

**Comment by slhenry** [comment]
EOWG minutes http://www.w3.org/2015/12/18-eo-minutes (starting at "Brent: Getting Started Tips")

**Comment by slhenry** [comment]
[mild] wording & punctuation suggestion:
"Provide space in your design for alternatives for images and media as needed."
-> "Provide a place in your design for alternatives for images and media, as needed."

**Comment by slhenry** [comment]
Comments on the Example
* [medium] suggest "Example: Design shows alternative format options"
                            -> "Example: Design includes links to transcript and alternative video"
* [strong] The teddy bear is way to visually distracting. Please get rid of it.
* [medium-strong] Separate the links "Transcript Video with audio description" -- maybe just with a comma

**Comment by davidberman** [comment]
[mild] wording and punctuation suggestion:
"Provide space in your design for alternatives for images and media as needed."
-> "Provide a place in your design for alternatives for images and media"

**Comment by James-Green** [comment]
Layout bug on Safari 9.1 OS X 10.11.4
![screen shot 2016-04-07 at 2 47 41 pm](https://cloud.githubusercontent.com/assets/13589074/14364736/d2bc4b76-fccf-11e5-8c53-3c6ef9c12dbf.png)

**Comment by yatil** [comment]
@James-Green: That looks like a Safari bug to me, if I reset the vertical-align value in the Web inspector, the AD symbol is positioned correctly… Needs investigation. *goes opens another issue*

**Comment by sharronrush** [comment]
This is a residual comment from a since published resource, if there is no more discussion, please close.
